scripts/ci/docker/generate-tag.mjs

```
@@ -114,7 +121,10 @@ function getTypeOfDeployment() {
       prod: true,
     }
   }
-
+  return {
+    dev: true,
+    prod: false
+  }
   throw new Error(`Unsupported branch: ${targetBranch}`)
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)